### ISSUE TITLE
If the barrier extension (FHIR-36691) is on the goal, it needs to be displayed

### ISSUE THREAD

**Issue by kbertodatti** [chronic-care/mcc-project#115]

**Comment by kbertodatti** [comment]
@swmuir to check if this is there.

**Comment by kbertodatti** [comment]
@swmuir @drdavec do you mind checking if this is in MyCarePlanner?

**Comment by drdavec** [comment]
@kbertodatti No, it is not displayed.  No extensions are displayed, only basic fields for description, author (expressed by), start date, and due date.  We have no way to create or edit any other fields, so nothing to display.  And none of these other fields are returned by any EHR system.

**Comment by kbertodatti** [comment]
Hmm ok - Would this be icebox or delete then?

**Comment by xmlmodeling** [comment]
hmm, these barriers were of interest to Emma.  but we have no plans or
scenarios to capture at this time.  Still relevant though.  So, Icebox?
